src/index.js

```diff
@@ -67,12 +70,16 @@ const init = async () => {
   });
 
   await server.start();
-  console.log("Server running on %s", server.info.uri);
+  console.log(
+    "Server running on http://localhost:3000/eligibility-checker/grant-name/start",
```

Comment:
we're using `eligibility-checker` a lot in different hard coded values. This would be better driven through `config/app.js` and composed.

Reply:
Will be part of your refactor